bin/graftM

```diff
@@ -129,60 +132,59 @@ if __name__ == '__main__':
                                  (Run.MIN_ALIGNED_FILTER_FOR_NUCLEOTIDE_PACKAGES, Run.MIN_ALIGNED_FILTER_FOR_AMINO_ACID_PACKAGES))
 
     searching_options = graft_parser.add_argument_group('searching options')
-    searching_options.add_argument('--evalue', metavar='evalue', help='evalue cutoff for the hmmsearch (default = 1e-5)', default= '1e-5')
-    searching_options.add_argument('--search_and_align_only', action="store_true", help='Stop after reads have been identified (default=False)', default=False)
-    searching_options.add_argument('--search_only', action="store_true", help='Stop after reads searched (default=False)', default=False)
-    searching_options.add_argument('--euk_check', action="store_true", help='Search whole sample for 18S (default=False)', default=False)
-    searching_options.add_argument('--search_method', choices=('hmmsearch','diamond'), help='Search method (default=hmmsearch)', default='hmmsearch')
-    searching_options.add_argument('--maximum_range', type=int, help='maximum range to use when searching for potentially linked reads (when searching contigs)', default=None)
-    searching_options.add_argument('--expand_search_contigs', nargs='+', help='generate an HMM for searching --forward data by first searching these contigs')
+    searching_options.add_argument('--evalue', metavar='evalue', help='Specify the evalue cutoff for the hmmsearch, if you would like to use a cutoff different to the default (1e-05) or the trusted cutoff (TC) within the HMM.', default= '1e-5')
+    searching_options.add_argument('--search_and_align_only', action="store_true", help='Stop GraftM running after reads have been identified and aligned (i.e. no placement step, default=False)', default=False)
+    searching_options.add_argument('--search_only', action="store_true", help='Stop GraftM running after reads have been identified (i.e. no alignment or placement steps, default=False)', default=False)
+    searching_options.add_argument('--euk_check', action="store_true", help='Cross check identified reads using an 18S specific HMM to help filter out eukaryotic ribosomal reads (default=False)', default=False)
+    searching_options.add_argument('--search_method', choices=('hmmsearch','diamond'), help='Search method, either hmmsearch or DIAMOND (default=hmmsearch)', default='hmmsearch')
+    searching_options.add_argument('--maximum_range', type=int, help='Maximum range to use when searching for potentially linked reads (when searching contigs)', default=None)
+    searching_options.add_argument('--expand_search_contigs', nargs='+', help='Provide an alignment of the sample being searched. This alignment will initially be searched for full length genes, from which a sample specific HMM model will be created and used in the search step of graftM.')
```

Comment:
provide an _assembly_

This _assembly_

used as an additional HMM in the search step.